### PR TITLE
[sflow] Added an argument to enable Sflow feature on DUT

### PR DESCRIPTION
**Pull request by AndriiLozovyi**
Signed-off-by: Andrii-Yosafat Lozovyi <andrii-yosafatx.lozovyi@intel.com>

### Description of PR

Summary:
Sflow test suite checks if sFlow feature is enabled on DUT, and if not then skips all tests. 
By default, sFlow is disabled on DUT -  [init_cfg](https://github.com/Azure/sonic-buildimage/blob/afee1a851c5e96c54bfb933bd82fe369df932a88/files/build_templates/init_cfg.json.j2#L36),  [config](https://github.com/Azure/sonic-buildimage/blob/9e90fac18b3b741bc5dedc4a9d37ec423c0304dd/rules/config#L103).
So in order to run this tests automatically without manually turning sFlow feature, I added argument which can be passed to test to turn Sflow feature on DUT. 

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [x] Test case(new/improvement)

#### What is the motivation for this PR?
Be able to run test_sflow cases automatically by passing an argument to TC, without manually changing sFlow feature status on DUT before TC run. 
#### How did you do it?
Added a fixture that enables sFlow feature on DUT by passing an argument.
#### How did you verify/test it?
Run test_sflow with --enable_sflow_feature argument, test cases wasn't skipped.
#### Any platform specific information?
```
SONiC.master.88-dirty-20210108.105001
Distribution: Debian 10.7
Kernel: 4.19.0-9-2-amd64
Build commit: dbc67184
Build date: Fri Jan 8 10:59:56 UTC 2021
Platform: x86_64-arista_7170_64c
HwSKU: Arista-7170-64C
```

### CODE REVIEW

**Comment by SavchukRomanLv** [comment]
@yxieca please take a look

**Comment by SavchukRomanLv** [comment]
@wangxin can you please take a look? Thanks in advance

**Review by wangxin — APPROVED** [review]

File: tests/sflow/test_sflow.py [hunk 1]
```diff
@@ -106,7 +113,14 @@ def get_ifindex(duthost, port):
...
     duthost.shell('config sflow %s'%sflow_status)
     time.sleep(2)
+# ----------------------------------------------------------------------------------
 
+@pytest.fixture(scope='module')
+def config_sflow_feature(request, duthost):
+    # Enable sFlow feature on DUT if enable_sflow_feature argument was passed
+    if request.config.getoption("--enable_sflow_feature"):
+        duthost.shell("sudo config feature state sflow enabled")
+        time.sleep(2)
```
Comment (yxieca) [inline tests/sflow/test_sflow.py:123]: Can you check that sflow is not enabled before enabling it? Also if the fixture enabled the sflow feature, can you disable it after the test is done?

Comment (AndriiLozovyi) [reply tests/sflow/test_sflow.py]: @yxieca I added check if sflow is not enabled, at teardown there is config_reload which will disable sflow feature even if I enabled it.

**Pull request merged**